src/triggers/default_triggers.py

```diff
@@ -32,7 +32,11 @@
             vip_dhcp_allocation=False,
             master_memory=resources.DEFAULT_MASTER_SNO_MEMORY,
             master_vcpu=resources.DEFAULT_MASTER_SNO_CPU,
-            network_type=consts.NetworkType.OVNKubernetes,
+            network_type=None,
+        ),
+        "let_service_choose_network_type": Trigger(
+            condition=lambda config: version.parse(config.openshift_version) >= version.parse("4.12.0-0.0"),
```

Comment:
apparently the parsing of ``4.12.0-0.0`` produces the following warning:
```
packaging/version.py:111: DeprecationWarning: Creating a LegacyVersion has been deprecated and will be removed in the next major release
```
(see https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_assisted-test-infra/1867/pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-4-12/1576609548653826048#1:build-log.txt%3A581)

I think ``config.openshift_version`` is either way only a prefix for the selected version (e.g. ``4.11``, ``4.12``, etc.) so maybe we should change it to:
```suggestion
            condition=lambda config: version.parse(config.openshift_version) >= version.parse("4.12"),
```